addons/website_event_questions/models/event_registration.py

```diff
@@ -28,3 +28,10 @@ class EventRegistrationAnswer(models.Model):
     _sql_constraints = [
         ('value_check', "CHECK(value_answer_id IS NOT NULL OR COALESCE(value_text_box, '') <> '')", "There must be a suggested value or a text value.")
     ]
+
+    # for displaying selected answers by attendees in attendees list view
+    def name_get(self):
+        res = []
+        for rec in self:
+            res.append((rec.id, ("%s" % rec.value_answer_id.name)))
```

Comment:
Well, shouldn't we either take value_answer_id or value_text_box ? I don't remember all details of this model, but if I remember well either this is predefined, either user entry ? In that case, how is it displayed with your name_get ?

Reply:
we only need to take value_answer_id, in the specification it mentioned that no need to display the text input in the list view.